functions/index.js

```diff
@@ -198,6 +86,8 @@ exports.sendPushNotification = functions.https.onRequest(async (req, res) => {
 
   docData.totalCount = docData.totalCount + 1;
 
+  console.log('Sending payload', JSON.stringify(payload));
```

Comment:
Let's not log in production. That means we would end up logging all notifications.